### ISSUE TITLE
Authors can't see the options of wright right-click on the abstract form!

### ISSUE THREAD

**Issue by aymeniq** [pkp/pkp-lib#10845]
Hi,

I'm hearing from our authors that they are unable to access the right-click options when filling out the abstract form. Our authors report that when they try to right-click on the abstract field, they don't see any context menu or options appear. This is making it very difficult for them to perform basic tasks like copying and pasting text into the abstract. They can simply paste the text of the abstract using ctr+v, but they can't do it by right-clicking and choosing the paste option. They can do it right on other forms like title, and keywords! Any guidance or advice you can offer to resolve this problem would be greatly appreciated.

![Image](https://github.com/user-attachments/assets/67534f1a-2d61-4625-99eb-f784f2b7d803)

Kindest regards,

Aymen

**Comment by asmecher** [comment]
@aymeniq, I see you've already posted this on our support forum:

https://forum.pkp.sfu.ca/t/authors-unable-to-access-right-click-options-on-abstract-form/94682

I'll close this in favour of the forum. It's best not to post the same question in multiple places.
